### DISCUSSION TITLE
help:how i add_jobs in docker?

### DISCUSSION THREAD

**Discussion by gaozy09** [cybertec-postgresql/pg_timetable#283] [Q&A]
please!
docker run it like this:
docker run --name pg_timetable cybertecpostgresql/pg_timetable:latest

but i how to timetable.add_job in docker?

**Comment by gaozy09** [comment]
else

my database postgres is also in docker

**Comment by pashagolub** [comment]
Hello,

since **pg_timetable** is controlled by PostgreSQL, to create jobs you need the ability to connect to the target database with your preferred client, e.g. `psql`.

You can check [samples](https://github.com/cybertec-postgresql/pg_timetable/tree/v4_dev/samples) folder for examples. Usually, I'm using a command like this:
```bash
$ psql -U scheduler -d timetable -c "\i samples/Autonomous.sql"
```

However, you may use whatever SQL client you prefer. We are working hard on a GUI application to manage jobs, tasks, and other meta for **pg_timetable**. We hope to release the first public version by the end of July.

Regards

**Reply by gaozy09** [reply]
thanks
